packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -785,7 +785,16 @@ void main() {
                 ),
               ],
             ),
-            TestSemantics(),
+            TestSemantics(
+              actions: <SemanticsAction>[SemanticsAction.tap],
```

Comment:
This is a bit surprising, why there is one parent node and one child node? This may cause an extra swipe to reach the node with 'Scrim' label.

Reply:
I imagine there should be just one node with tap and 'Scrim' label.

Reply:
Thanks! Removed onTap from parent node.